samples/java-protobuf-doc-snippets/src/it/java/com/example/JwtIntegrationTest.java

```diff
@@ -58,7 +58,8 @@ public void testMsgWithClaim() throws Exception {
 
   private String bearerTokenWith(Map<String, String> claims) throws JsonProcessingException {
     // setting algorithm to none
-    String alg = Base64.getEncoder().encodeToString("{\"alg\":\"none\"}".getBytes()); // <4>
+    String alg = Base64.getEncoder()
+        .encodeToString("{\"alg\":\"none\"}".getBytes()); // <4>
```

Comment:
We are still using java `11` for java protobuf samples.. so no blocks available here.

Reply:
oh, it's a pity